src/components/pages/download/MacTab.astro

```diff
@@ -25,3 +55,21 @@ import Tab from "./Tab.astro";
         </p>
     </Card>
 </Tab>
+
+<style>
+    .seqouia-summary {
+        cursor: pointer;
+        font-weight: bold;
+    }
+
+    .sequoia-install-img {
+        margin-top: 1rem;
+        border-radius: 0.5rem;
+    }
+
+    @media (max-width: 1300px) {
+        .sequoia-install-img {
+            width: 50vw;
+        }
+    }
```

Comment:
do we really need this? usually `img { max-width: 100%; }` does the trick

Reply:
accidentally left this in while i was testing different styles